rtcstats.js

```diff
@@ -32,7 +32,7 @@
     // TODO: drop getStats when not connected?
     if (connection.readyState === 1) {
       connection.send(JSON.stringify(arguments));
-    } else {
+    } else if (arguments[0] !== 'getstats') {
```

Comment:
a good limit value is rather hard since (assuming this is going to use the same Q) the number of peerconnection calls (in particular onicecandidate + addIceCandidate) can be quite high.